nucypher/blockchain/eth/sol/__conf__.py

```diff
@@ -15,4 +15,4 @@
 along with nucypher.  If not, see <https://www.gnu.org/licenses/>.
 """
 
-SOLIDITY_COMPILER_VERSION = '0.6.6'
+SOLIDITY_COMPILER_VERSION = 'v0.6.7'
```

Comment:
Just curious: why the leading `v`?

Reply:
Just a slight difference in how we `py-solc-x` handles inputted version strings.